src/library/columncache.h

```diff
@@ -55,6 +55,7 @@ class ColumnCache : public QObject {
         COLUMN_LIBRARYTABLE_COVERART_TYPE,
         COLUMN_LIBRARYTABLE_COVERART_LOCATION,
         COLUMN_LIBRARYTABLE_COVERART_HASH,
+        COLUMN_LIBRARYTABLE_ANALYZE_PATH,
```

Comment:
I just noticed this. The term "LIBRARYTABLE" refers to our `library` table. This is misleading, because all other columns are actually part of this table.

How about `COLUMN_REKORDBOX_ANALYZE_PATH`? It should also be placed after the columns of the tables that belong to the internal track collection. The same applies to the string constants in dao/trackschema.h

Reply:
In the long-term we need to rethink the design of `ColumnCache`, because it should be independent of any optional, concrete features.